.travis.yml

```diff
@@ -28,6 +32,12 @@ matrix:
     env:
       - CONFIGURATION=linux/arm64
 
+  - os: linux
+    arch: amd64
+    dist: focal
```

Comment:
While you are hot on the subject, please send a PR for 20.10 `groovy` so we are ready for next month. :-)